cwms_radar_api/src/main/java/cwms/radar/security/CwmsAccessManager.java

```diff
@@ -192,8 +188,7 @@ public String getName() {
 
     @Override
     public boolean canAuth(Context ctx, Set<RouteRole> roles) {
-        return  (ctx.cookie("JSESSIONIDSSO") != null)
-              ||(ctx.cookie("JSESSIONID") != null);
+        return ctx.cookie("JSESSIONIDSSO") != null;
```

Comment:
"JSESSIONIDSSO" should be a constant somewhere.  too easy to typo or mix up with "JSESSIONID"

Reply:
I know its not in your changes but the "office" should probably use the Controllers constant - might make it a little clearer that its hoping that parameter is present.

Reply:
good point, especially until we clear that up.